<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientEdit</class>
 <widget class="QDialog" name="ClientEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="clientFirstNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="clientLastNameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Паспорт</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="clientPassportEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="dealHistory">
     <property name="title">
      <string>История страховок</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTableView" name="dealView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Действитедьна в промежутке</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>начиная с</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="dealFilterAfterDate"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>заканчивая до</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dealFilterBeforeDate"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="filterApply">
           <property name="text">
            <string>Применить фильтр</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="filterClear">
           <property name="text">
            <string>Очистить фильтр</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="createNewDealButton">
        <property name="text">
         <string>Заключить новую сделку</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClientEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClientEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
